### DISCUSSION TITLE
feat: add NZBGet support

### DISCUSSION THREAD

**Discussion by josh1703658784** [autobrr/autobrr#1855] [Ideas]
NZBGet is a great alternative to SABnzbd that is fully featured.

It would be great to have support for NZBGet added. For a while the project seemed stale but it is in active development these days.

If there's any additional detail I can provide please let me know.

Related: [Issue 543](https://github.com/autobrr/autobrr/pull/543)

**Comment by KyleSanderson** [comment]
The NZBGet* project was archived in 2022 - is there a new area that's seeing active development?

**Comment by darthShadow** [comment]
Yeah, the development has been moved over to https://github.com/nzbgetcom/nzbget, which has seen a steady stream of releases.

FWIW, the LSIO docker image was also revived using the same fork: https://github.com/linuxserver/docker-nzbget#versions

**Comment by zze0s** [comment]
Added in https://github.com/autobrr/autobrr/pull/2370
